### DISCUSSION TITLE
Deep Dive: Astroid Boilerplate Site 206

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#848] [Video Tutorials]
New Article and linked Video are here:
Deep Dive: Astroid Boilerplate Site 206
https://astroidframe.work/blog/astroid-hands-on
Probably at least one if not two more in this current series...

Please like, share, comment etc.

And help get the word out that Astroid Rocks.

Any ideas for future tutorials?
A future Series?

a) Article Layout Builder Series?
b) Multi Language Site Series (I believe that Joomla 5.2 has new multilingual features?!?)
c) Series XYZ???

Thanks and Enjoy!

**Comment by sonvnn** [comment]
Hi @paurray 

Thank you so much for your contribution. I think Article Layout is an interesting topic that users want to experience at the moment. It provides administrators with many ways to present content at many levels. It would be great if you could make a video tutorial to guide users to do it. 

If you have any questions, please contact me directly or on Discussion. I will be happy to answer any questions so that you can complete your great products.

Thanks & Best regards,
Sonny
